src/System.Runtime/src/System/HashCode.cs

```diff
@@ -0,0 +1,406 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace System
+{
+    public struct HashCode
+    {
+#   if SYSTEM_HASHCODE_TESTVECTORS
+        private static readonly uint _seed = 0;
+#   else
+        private static readonly uint _seed = unchecked((uint)new Random().Next(int.MinValue, int.MaxValue));
+#   endif
+
+        private const uint Prime1 = 2654435761U;
+        private const uint Prime2 = 2246822519U;
+        private const uint Prime3 = 3266489917U;
+        private const uint Prime4 = 0668265263U;
+        private const uint Prime5 = 0374761393U;
```

Comment:
* I've been working with the C# language (and .NET) for more than 13 years now.
* Today I learned that a `0` prefix doesn't mean octal.

Together, to me, these points say "please remove the leading 0".

(My first thought (given the knowledge that it had compiled) was a quirky behavior that since an 8 and 9 were present that it knew it couldn't be octal, but apparently C# simply removed that confusing part of C, but without removing the "confusing" flag from my head.)